fieldservice_stock/models/fsm_equipment.py

```diff
@@ -27,6 +27,12 @@ def _compute_current_stock_loc_id(self):
             else:
                 equipment.current_stock_location_id = False
 
+    @api.onchange('product_id')
+    def _onchange_product(self):
+        for equipment in self:
+            # self.lot_id = False
```

Comment:
@marcelsavegnago - Do you have a particular reason to leave out the lot_id?

Reply:
@brian10048 I don't really remember the reason but I believe that it is because the lot_id field is a required field and when assigning the value of False it gave a problem. Do you have any suggestions ?

Other than that, I forgot to remove the commented line: D